src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import './variables';
+@import './stopwatch';
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when importing SCSS files, you should not include the file extension. Remove the '.scss' from the import statements.